core/dbt/contracts/graph/manifest.py

```diff
@@ -944,6 +934,19 @@ def __reduce_ex__(self, protocol):
         return self.__class__, args
 
 
+class MacroManifest(MacroMethods):
+    def __init__(self, macros, files):
+        self.macros = macros
+        self.files = files
+        self.metadata = ManifestMetadata()
+        # Something is acessing the flat_graph in the macro_manifest.
```

Comment:
is this something to look further into?

Reply:
It's working, but that's pretty kludgy.  Need to find what code is calling that and prevent it if it's a MacroManifest. On the list.

Reply:
It's returned by the 'graph' context property in the ProviderContext class. I think that preventing that would probably make more of a mess than just leaving the attribute there, i.e. not worth it to remove.

Reply:
Is this in the sense of, user's ability to accessing `graph.macros` from the [Jinja context](https://docs.getdbt.com/reference/dbt-jinja-functions/graph/)?

Reply:
This is just for the MacroManifest, which is only used in parsing. The flat_graph was always set to an empty dictionary in the macro manifest.